iocore/hostdb/test_RefCountCache.cc

```diff
@@ -264,7 +272,20 @@ main()
   // printf("Sync return: %d\n", cache->sync_all());
 
   printf("TestRun: %d\n", ret);
-  exit(ret);
+
+  delete cache;
+
+  return ret;
+}
+
+int main() {
+  int ret = test();
+
+  for (const auto item: ExampleStruct::items_freed) {
+    printf("really freeing: %p\n", item);
+    ::free(item);
+  }
+  ExampleStruct::items_freed.clear();
```

Comment:
Since this is a test, how about verifying that all the expected items were freed?